content/source/docs/plugin/framework/diagnostics.html.md

```diff
@@ -24,8 +40,10 @@ practitioners to fix their configuration or environment more quickly. You
 should only append to Diagnostics slices and never replace or remove
 information from them.
 
-~> **Important:** The design of diagnostics is expected to change to something
-less verbose in the near future.
+The next section will detail the properties and typical behaviors of
+diagnostics, while the final section will outline the typical methods for
+working with diagnostics, using functionality from the available
+[`diag` package](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/diag).
 
 ## Diagnostic Properties
```

Comment:
Does it make sense to have this anymore? We don't really _have_ a diagnostic type anymore to have these properties. Maybe we want to reword some of this to be about the _parts_ of a diagnostic?

Reply:
I really did like some of the additional context provided in there from your original verbiage -- I'll take a shot a paring it down a little given the new documentation.